modules/logging-enabling-loki-alerts.adoc

```diff
@@ -12,14 +12,14 @@ The `AlertingRule` CR contains a set of specifications and webhook validation de
 * If an `AlertingRule` CR includes an invalid `for` period, it is an invalid alerting rule.
 * If an `AlertingRule` CR includes an invalid LogQL `expr`, it is an invalid alerting rule.
 * If an `AlertingRule` CR includes two groups with the same name, it is an invalid alerting rule.
-* If none of above applies, an alerting rule is considered valid.
+* If none of the above applies, an alerting rule is considered valid.
 
 [options="header"]
 |================================================
 | Tenant type    | Valid namespaces for `AlertingRule` CRs
-| application    |
 | audit          | `openshift-logging`
-| infrastructure | `openshift-/\*`, `kube-/\*`, `default`
+| infrastructure | `openshift-\*`, `kube-*`, `default`
```

Comment:
```suggestion
| infrastructure | `openshift-*`, `kube-*`, `default`
```

Reply:
Ok, I'm not sure what it is escaping then, because if it's the `*` we should need one for the second occurrence as well. I'm also not sure if the current preview is up-to-date with the state of the PR.